.github/workflows/gradle.yml

```diff
@@ -8,7 +8,54 @@ on:
       - "gitbook/v1"
 
 jobs:
-  ## Gradle Build (Connectors Base)
+  # COMMON TASKS
+  ensure-images-exist:
+    name: "Ensure all required Docker images exist on Dockerhub"
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@v2
+
+      - name: Check images exist
+        run: ./tools/bin/check_images_exist.sh all
+  # The output of this job is used to trigger the following builds.
+  changes:
+    name: "Detect Modified Files"
+    runs-on: ubuntu-latest
+    outputs:
+      backend: ${{ steps.filter.outputs.backend }}
+      build: ${{ steps.filter.outputs.build }}
+      cli: ${{ steps.filter.outputs.cli }}
+      frontend: ${{ steps.filter.outputs.frontend }}
+    steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@v2
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
+            backend:
+              - 'airbyte-**/**'
```

Comment:
isn't this everything? including `airbyte-integrations`?

Reply:
yes, good call out. I'll tackle the airbyte-integration filtering in a follow up PR!